Lua/syntax_test_lua

```diff
@@ -0,0 +1,14 @@
+-- SYNTAX TEST "Lua.sublime-syntax"
+
+local func = function(arg1, arg2)
```

Comment:
You cleverly avoided the comma in-between the parentheses. The `,` also has the `variable.parameter` scope, which it shouldn't have.

Reply:
Well it currently works like this with every other function arguments by the default syntax. All the parameters and the , are colored the same way under the same scope.

Reply:
Fair enough. You see, [the whole syntax needs a facelift](https://github.com/sublimehq/Packages/issues/481). So I'm trying to figure out what this PR means. Is it a patch for this old crufty syntax, or is it the beginning of a series of changes that will eventually bring Lua up to standards? The choice is yours ;)

Reply:
Well that's a good question, i use a custom Lua syntax that i made, and i just wanted to fork some of the changes/fixes i made into the official one. The full rework probably needs a lot more work.